### ISSUE TITLE
FutureWarning received regarding boolean inputs to the 'inclusive' argument

### ISSUE THREAD

**Issue by lcorrigan** [man-group/dtale#579]
I'm starting dtale 1.57 in Python 3.8.10 through the VSCode interactive window for a dataframe that has the following dtypes: datetime64[ns](3), float64(6), object(5), string(1), uint16(5), uint64(2)

I'm not sure if this is a user-intended warning, as I can't determine what I passed in to make it state this message, but I receive multiple copies of this message after the call to dtale.show(data=mydataframe)

```
C:\Users\lcorrigan\AppData\Local\Programs\Python\Python38\lib\site-packages\dtale\utils.py:502: FutureWarning:

Boolean inputs to the `inclusive` argument are deprecated infavour of `both` or `neither`.
```

Wouldn't mind some guidance on this, especially if this is going to block me in some way I've yet to experience (calcs, plotting etc).

**Comment by aschonfeld** [comment]
@lcorrigan good catch! including an update for this in the next release. I don't think it should affect anything at the moment.  Pandas just changed their implementation of the parameter.  Regardless, the new version of D-Tale should be out within the next few days. I will keep you posted.

**Comment by aschonfeld** [comment]
Fixed in v1.58.1
